lighthouse-core/gather/gatherers/fonts.js

```diff
@@ -138,27 +172,42 @@ class Fonts extends Gatherer {
     });
   }
 
-  afterPass({driver}) {
+  /**
+   * @param {LH.Gatherer.PassContext} passContext
+   * @return {Promise<LH.Artifacts['Fonts']>}
+   */
+  afterPass(passContext) {
+    const driver = passContext.driver;
     const args = JSON.stringify(fontFaceDescriptors);
-    return Promise.all(
+    /** @type {Promise<[Array<LH.Artifacts.Font>, Array<LH.Artifacts.Font|FontGatherError>]>} */
+    const fontData = Promise.all(
       [
         driver.evaluateAsync(`(${getAllLoadedFonts.toString()})(${args})`),
         driver.evaluateAsync(`(${getFontFaceFromStylesheets.toString()})()`),
       ]
-    ).then(([loadedFonts, fontFaces]) => {
-      return loadedFonts.map(fontFace => {
-        if (fontFace.err) {
```

Comment:
as far as I can tell, this was a bug. `fontFace` came from `getAllLoadedFonts` while it was `getFontFaceFromStylesheets` that generates the error objects it's looking for